draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -1410,6 +1420,28 @@ Content-Type: text/plain
    this check, but &MUST; provide a setting which enables it.
 </t>
 </section>
+<section title="IP-ID reference identity" anchor="https.ip-id">
+<t>
+   A server that is identified using an IP address literal in the "host" field
+   of an "https" URI has a reference identity of type IP-ID.  An IP version 4
```

Comment:
I note that in the recently approved draft-ietf-dtn-tcpclv4, we used the term IPADDR-ID for the iPAddress SAN.
If we think that IP-ID is better, I can try to reach out to get the DTN doc changed during AUTH48 to be consistent.

Reply:
Thanks!